pyproject.toml

```diff
@@ -1,15 +1,202 @@
 [build-system]
-build-backend = "setuptools.build_meta"
-requires = ["setuptools"]
+build-backend = "hatchling.build"
+requires = [
+  "hatchling",
+]
+
+[project]
+name = "servestatic"
+description = "Production-grade static file server for Python web apps."
+readme = "README.md"
+keywords = [
+  "Django",
+]
+license = "MIT"
+authors = [
+  { name = "Mark Bakhit" },
+]
+requires-python = ">=3.9"
+classifiers = [
+  "Development Status :: 5 - Production/Stable",
+  "Framework :: Django",
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: MIT License",
+  "Operating System :: OS Independent",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Topic :: Communications :: File Sharing",
+  "Topic :: Internet :: WWW/HTTP",
+  "Topic :: Internet :: WWW/HTTP :: WSGI :: Middleware",
+]
+dynamic = [
+  "version",
+]
+dependencies = [
+  "django",
+]
+optional-dependencies.brotli = [
+  "brotli",
+]
+urls.Changelog = "https://archmonger.github.io/ServeStatic/latest/changelog/"
+urls.Documentation = "https://archmonger.github.io/ServeStatic/"
+urls.Homepage = "https://github.com/Archmonger/ServeStatic"
+
+[tool.hatch.version]
+path = "src/servestatic/__init__.py"
+
+[tool.hatch.build.targets.sdist]
+include = [
+  "/src",
+  "/tests",
+]
+
+[tool.hatch.build.targets.wheel]
+nclude = [
+  "/src",
+  "/tests",
+]
+
+[tool.hatch.metadata]
+license-files = { paths = [ "LICENSE" ] }
+
+[tool.hatch.envs.default]
+installer = "uv"
+
+#
+# Testing
+#
+[tool.hatch.envs.hatch-test]
+extra-dependencies = [
+  "pytest-sugar",
+  "requests",
+  "brotli",
+]
+randomize = true
+matrix-name-format = "{variable}-{value}"
+
+# Django 3.2 LTS
+[[tool.hatch.envs.hatch-test.matrix]]
+python = [ "3.8", "3.9", "3.10"]
```

Comment:
We dropped Python 3.8 support with `v2.0.0` in order to simplify some of our async code.

Is there any point to calling out `3.8` within this matrix?

Reply:
It's mostly for completeness against https://docs.djangoproject.com/en/5.1/faq/install/ but I can remove it entirely

Reply:
Yeah let's remove it. Keeping it there can imply that if the user manually runs this test environment then it should work.